Source/DatabaseManager/DatabaseManager/Game.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DatabaseManager
+{
+    public class Game
+    {
+#nullable disable
+        public int Id { get; set; }
+        public DateTime CreationTime { get; set; }       
+        public bool GameEnded { get; set; }

```

Comment:
Instead of GameEnded as a property you can have a public boolean to see if there is a player which have won (all pieces in goal square).